plugins/wazuh-core/server/services/initialization/initialization.ts

```diff
@@ -0,0 +1,171 @@
+import { Logger } from 'opensearch-dashboards/server';
```

Comment:
Separate classes in several files

Reply:
The classes were separated in different files.